### DISCUSSION TITLE
Radiation fraction in large pool fires

### DISCUSSION THREAD

**Discussion by hsaglik** [firemodels/fds#13832] [Q&A]
Hi. 
I modelled a gasoline pool fire with a fire diameter of about 12 m.

As the pool fire's diameter increases, the radiation fraction decreases up to 0.2. My question starts at this point.
I defined the fraction of radiation as 0.27. Consequently, I obtained the ratio of Q_RADI/Q_TOT is about 0.14 from analysis at steady-state. I know that heavy smoke can absorb radiation. However, the ratios for the fraction of radiation have been determined experimentally. Thus, these observations should include the smoke layer. 

In this way, it seems to me that I should increase the input value of the fraction of radiation in FDS until I get the ratio of about 0.2 by Q_RADI/Q_TOT. Do you think this approach is true?

**Comment by rmcdermo** [comment]
FDS applies the radiation loss fraction to the flame volume only.  So any H2O in the atmosphere between the flame and the boundary absorbs and decreases the Q_RADI reported in the _hrr file.  So, yes, I think it is reasonable to dial up your CHI_R to match the experimentally obtained value as long as the boundary is close to where your heat fluxes were measured.  Otherwise, I would put heat flux gauges at those locations and tune CHI_R based on those direct measurements instead of the radiant fraction.

**Reply by hsaglik** [reply]
Thank you for your answer.
Actually, I found the value of radiation fraction from an expression given based on some "experiments": 0.35*exp(-0.05D) [1]. I will not perform any test. 
I think, I got my answer, thank you so much!

1. McGrattan, K. B., Baum, H. R., & Hamins, A. (2000). Thermal radiation from large pool fires. National Institute of Standards and Technology. US Department of Commerce. USA.

**Comment by drjfloyd** [comment]
You may also want to consider adjusting the path length for the radiation
solver.

**Comment by Er9y714** [comment]
Relevant presentation about modelling large pool fires
https://www.femtc.com/events/2022/d1-02-mcgrattan/

**Comment by mcgratta** [comment]
If you want the `RADIATIVE_FRACTION` to be exactly as you specify, set `OPTICALLY_THIN=T` on the `RADI` line. However, this setting has the effect of making all smoke and other gases non-absorptive of thermal radiation. For some applications, this might be OK, but not for, say, a flashed over compartment or some other scenario where radiation absorption by gases is important.

**Comment by hsaglik** [comment]
@rmcdermo @mcgratta @drjfloyd @Er9y714 
Thank you so much for the tips!
